common/persistence/nosql/nosqlplugin/cassandra/shard.go

```diff
@@ -266,8 +252,6 @@ func (db *cdb) UpdateShard(ctx context.Context, row *nosqlplugin.ShardRow, previ
 		row.ClusterTimerAckLevel,
 		transferPQS,
 		transferPQSEncoding,
-		crossClusterPQS,
-		crossClusterPQSEncoding,
```

Comment:
UpdateShard can set these fields to nil/empty so the existing shard records get leaner. And then in a follow up PR get rid of these completely.
Alternatively we can write a script to do that. Up to you

Reply:
I want to do the persistence changes in a separate PR because they're going to also be quite large, but should't affect any application code changes. They're also high-risk to change, so I didn't want to add to an already quite big PR